src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/CodeBasedDependencyAlgorithm.cs

```diff
@@ -67,6 +67,15 @@ public static void AddDependenciesDueToMethodCodePresence(ref DependencyList dep
                         "UniversalCanon signature of method"));
                 }
 
+                if (method.IsVirtual && !method.HasInstantiation)
```

Comment:
If this is a dependency of the virtual invoke map, shouldn't it just be reported by the dependencies from the line below?

```csharp
dependencies.AddRange(ReflectionVirtualInvokeMapNode.GetVirtualInvokeMapDependencies(factory, method));
```